packages/config/config/devices/0x032e/dat-101.json

```diff
@@ -0,0 +1,400 @@
+{
+	"manufacturer": "Defaro",
+	"manufacturerId": "0x032e",
+	"label": "DAT-101",
+	"description": "Thermostat Actuator",
+	"devices": [
+		{
+			"productType": "0x0013",
+			"productId": "0x0021"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "State changes",
+			"maxNodes": 10,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "External temperature sensor",
+			"maxNodes": 1
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Relay mode",
+			"description": "Selects NO/NC modes",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal OPEN (default)",
```

Comment:
Don't mention `(default)` in labels, this is redundant with the `default` property